findmyparadise_frontend/src/components/Events.vue

```diff
@@ -62,7 +65,8 @@ export default {
   },
   mounted() {
     axios
-      .get("http://54.91.69.145:80/api/events")
```

Comment:
Did you mean to change the api?

Reply:
Ideally I need to have an api that takes in the listingId from props, but haven't figured it out yet. If the path I want is events/{listingId}/25, then would I do .get("http://54.91.69.145:80/api/events/15/25") or .get("http://localhost:8080/api/events/15/25")?

Reply:
Use the api with the ip address

Reply:
I sent commit with the proper route.